packages/eventual-send/src/E.js

```diff
@@ -31,13 +32,13 @@ const baseFreezableProxyHandler = {
  * A Proxy handler for E(x).
  *
  * @param {*} x Any value passed to E(x)
- * @param {import('./index').HandledPromiseConstructor} HandledPromise
+ * @param {import('./types').HandledPromiseConstructor} HandledPromise
  * @returns {ProxyHandler} the Proxy handler
  */
-function EProxyHandler(x, HandledPromise) {
-  return harden({
+const makeEProxyHandler = (x, HandledPromise) =>
+  harden({
     ...baseFreezableProxyHandler,
-    get(_target, p, receiver) {
+    get: (_target, p, receiver) => {
```

Comment:
What's the narrative here? Arrow functions introduce more ascii soup than object function style.

Reply:
This change reflects an aspiration to conform to the (evolving) [https://github.com/endojs/Jessie](Jessie syntax).

Ideally, as much Endo code as possible would be written in Jessie (or TinySES) to facilitate porting it to platforms that choose not to implement all of Javascript's foibles.

Reply:
@michaelfig 

> This change reflects an aspiration to conform to the (evolving) [Jessie syntax](https://github.com/endojs/Jessie#readme%5D).
> 
> Ideally, as much Endo code as possible would be written in Jessie (or TinySES) to facilitate porting it to platforms that choose not to implement all of Javascript's foibles.

I appreciate having this here for reference.

Reply:
@kungfooman did you have something in mind to add or is this resolved?

Reply:
Nothing to add, thank you both, I didn't know it was due to the limitation of the Jessie parser.

Reply:
Thanks @kungfooman!